src/modules/utils/Utils.ts

```diff
@@ -67,4 +67,38 @@ export class Utils
         }
         return str;
     }
+
+    /**
+     * It is a function that outputs an object with BigInt as JSON.
+     * @param data - Object to be converted to a JSON string
+     */
+    public static toJson(data: object): string
+    {
+        if (data === undefined)
+            return "";
+
+        let count1 = 0, count2 = 0;
```

Comment:
This code is so unusual that I think it needs an explanation.

Reply:
I added some document, Is this enough?